test/chat_api/conversations_test.exs

```diff
@@ -709,6 +709,51 @@ defmodule ChatApi.ConversationsTest do
     end
   end
 
+  describe "list_conversations_by_account_paginated/1" do
+    test "sorts the conversations by most recent activity" do
+      account = insert(:account)
+      filter = %{"limit" => 5}
+      base_last_activity_at = DateTime.from_naive!(~N[2020-12-01 00:00:00], "Etc/UTC")
+
+      [third_batch, second_batch, first_batch] =
+        15..1
+        |> Enum.map(
+          &insert(:conversation,
+            account: account,
+            last_activity_at: DateTime.add(base_last_activity_at, &1 * 3600, :second)
+          )
+        )
+        |> Enum.chunk_every(5)
+
+      %{metadata: metadata1, entries: entries1} =
+        Conversations.list_conversations_by_account_paginated(account.id, filter)
+
+      assert Enum.map(entries1, & &1.id) == Enum.map(third_batch, & &1.id)
+
+      %{metadata: metadata2, entries: entries2} =
+        Conversations.list_conversations_by_account_paginated(
+          account.id,
+          Map.merge(filter, %{
+            "after" => metadata1.after
```

Comment:
`after` here is an encoded representation of the cursor pagination. The frontend needs to get a hold of this value, so that it can pass it on the request and get the next set of result